### ISSUE TITLE
Cannot register my students using csv file

### ISSUE THREAD

**Issue by grapenewt** [RunestoneInteractive/RunestoneServer#1458]
**What Course are you in**
Knowing the course will help us a lot
CSC157

**What Page were you on**
Manage Students: Register Students

**What is your username**
jwills

**Describe the bug**
A clear and concise description of what the bug is.  Vague statements like X does not work are not helpful.
When using a csv file to register students, I get a course does not exist error. I have been successful in the past. Not sure why this error is occurring now.

**Traceback**
If you got here through the Bug Report page and there was a stack trace, please paste it here

**Additional context**

**Comment by bnmnetp** [comment]
Have you checked to make sure you don’t have a typo in the class name somewhere?  Or maybe you accidentally have the fields in the wrong order?  I just used this myself and had no problems.

**Comment by grapenewt** [comment]
Yes. I have checked both of those. I have just tried signing a student up through the regular signup page and it was successful.
I have two sections of 24 students. So the csv file would be most helpful.
Thanks for your response.

**Comment by grapenewt** [comment]
Okay. When I used Macs Pages Program to create the CSV file it worked.
Using excel did not. 
Strange, but glad it worked.
Thanks again Brad for helping out. 
This will be the first time I am using runestone for an official course. Looking forward to it.
Thanks for all that you do for education.
Jim

**Comment by bnmnetp** [comment]
I would love to see a line from that excel generated CSV.  I just use a text editor when I create a csv so I wonder what unexpected stuff excel is adding.

**Comment by bnmnetp** [comment]
I wonder if Excel exported it but included column headers?

**Comment by grapenewt** [comment]
I thought of that. In fact, when I opened up the csv file it had the
first student line highlighted. Which means it may have been
interpreting the first line as a header. 

BTW. I tried using Pages again, and this time it complained I needed 6
fields, when in fact I had 6. Very strange. 

At any rate. In the end, I went with your suggestion of using a standard
text file. 

Worked perfectly. 

On 2020-06-06 6:16 pm, Bradley Miller wrote:

> **********************
> CAUTION: EXTERNAL MAIL
> ********************** 
> 
> I wonder if Excel exported it but included column headers? 
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
 

Links:
------
[1]
https://github.com/RunestoneInteractive/RunestoneServer/issues/1458#issuecomment-640130594
[2]
